pkgs/development/python-modules/qtile-bonsai/default.nix

```diff
@@ -0,0 +1,77 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  setuptools-scm,
+  cairocffi,
+  cffi,
+  strenum,
+  psutil,
+  xcffib,
+  pdm-backend,
+  pytestCheckHook,
+  pyside6,
+  pyvirtualdisplay,
+  qtile,
+  extraPackages ? [ ],
+}:
+
+buildPythonPackage rec {
+  pname = "qtile-bonsai";
+  version = "0.4.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "aravinda0";
+    repo = "qtile-bonsai";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-IWy/YEVdZc+UgIKl75ZpOkOIvpS5hCX0ihQenUOuJHo=";
+  };
+
+  build-system = [
+    setuptools
+    pdm-backend
+  ];
+
+  dependencies = extraPackages ++ [
+    strenum
+    psutil
+    pyside6
+    pyvirtualdisplay
+    setuptools
+    setuptools-scm
```

Comment:
These should be in `build-system` and not be normal dependencies usually

Reply:
you are right just removed them and works fine